# l, alpha: Polar coordinates of castor-platform joint in platform CS.
# d: Distance between above defined joint and wheel-ground contact point.
# r: Wheel radius.
# beta: Angle the castor normal makes with alpha.
# (As seen in chapter on castor wheels in ISBN 0-262-19502-X.)
float64 alpha
float64 beta
float64 d
float64 l
float64 r
